OsmAnd/src/net/osmand/plus/routing/data/AnnounceTimeDistances.java

```diff
@@ -189,13 +201,17 @@ private boolean isDistanceLess(float currentSpeed, double dist, double leadDist,
 	}
 
 	public float getSpeed(Location loc) {
-		boolean simulation = false;
-		if (locationProvider != null) {
-			simulation = locationProvider.getLocationSimulation().isRouteAnimating();
-		}
+		// Hardy 2023-08-05: I believe the limitation to the non-simulation case may be deprecated?
```

Comment:
too much comments just makes code worse to read

Reply:
Yes, true, these were just work notes for me ... :wink: - all removed now.